source/brailleTables.py

```diff
@@ -69,6 +77,33 @@ class BrailleTable(NamedTuple):
 and cleared when calling L{terminate}.
 """
 
+_inputTableForLangs = dict()
+"""Maps languages to input L{BrailleTable.fileName}."""
+
+_outputTableForLangs = dict()
+"""Maps languages to output L{BrailleTable.fileName}."""
```

Comment:
can you add types for these please, to note what types the dict holds

```suggestion
_inputTableForLangs: dict[str, str] = dict()
"""Maps languages to input L{BrailleTable.fileName}."""

_outputTableForLangs: dict[str, str] = dict()
"""Maps languages to output L{BrailleTable.fileName}."""
```